packages/constants/src/constants.ts

```diff
@@ -11,31 +11,65 @@ export const HIDE_LOADING = 'HIDE_LOADING';
 export const HIDE_ALL_PAGE_NOTIFICATIONS = 'HIDE_ALL_PAGE_NOTIFICATIONS';
 
 // Notifications
-export type TAppNotificationDomain = 'global' | 'page' | 'side';
-export const DOMAINS: {
-  [key: string]: TAppNotificationDomain;
```

Comment:
This was actually still wrong, as intellisense didn't provide any meaningful options.